core/NameHash.h

```diff
@@ -44,8 +44,12 @@ struct GlobalStateHash {
     UnorderedMap<NameHash, u4> methodHashes;
 };
 
+/** Sorts an array of NameHashes and removes duplicates. */
+void sortAndDedupe(std::vector<core::NameHash> &hashes);
```

Comment:
we try to not declare top-level functions.
Could this be a static function in some class?